pr-data.csv

```diff
@@ -1272,9 +1272,9 @@ https://github.com/apache/hive,54e43339dd671018fc70ebb5d9f0b292d70391a6,itests/l
 https://github.com/apache/hive,54e43339dd671018fc70ebb5d9f0b292d70391a6,itests/llap-tez,org.apache.hadoop.hive.llap.tezplugins.TestLlapTaskSchedulerService.testSimpleLocalAllocation,OD,,,
 https://github.com/apache/hive,54e43339dd671018fc70ebb5d9f0b292d70391a6,itests/llap-tez,org.apache.hadoop.hive.llap.tezplugins.TestLlapTaskSchedulerService.testSimpleNoLocalityAllocation,OD,,,
 https://github.com/apache/hive,54e43339dd671018fc70ebb5d9f0b292d70391a6,itests/llap-tez,org.apache.hadoop.hive.llap.tezplugins.TestLlapTaskSchedulerService.testUpdateWithError,OD,,,
-https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,standalone-metastore/metastore-server,org.apache.hadoop.hive.metastore.TestEmbeddedHiveMetaStore.testDatabase,ID,,,
-https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,standalone-metastore/metastore-server,org.apache.hadoop.hive.metastore.TestEmbeddedHiveMetaStore.testPartition,ID,,,
-https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,standalone-metastore/metastore-server,org.apache.hadoop.hive.metastore.TestEmbeddedHiveMetaStore.testSimpleFunction,ID,,,
+https://github.com/apache/hive,90fa9064f2c6907fbe6237cb46d5937eebd8ea31,standalone-metastore/metastore-server,org.apache.hadoop.hive.metastore.TestEmbeddedHiveMetaStore.testDatabase,ID,MovedOrRenamed,,
```

Comment:
The tests are actually from the superclass `TestHiveMetaStore` and it will be flaky when invoked from `TestEmbeddedHiveMetaStore`.
Thank Darko for finding that out, can you try yourself and rectify that?